tools/gulp/docs/__tests__/nestSections.test.js

```diff
@@ -56,8 +56,8 @@ describe('nestSections', () => {
         weight: 5
       },
       {
-        header: 'style',
-        reference: 'style'
+        header: 'design',
+        reference: 'design'
       },
       {
```

Comment:
I'm getting some errors here when running `yarn test` I think t's due to the change from `style` to `design` 

```  ● nestSections › sorts subpages by weight and header

    TypeError: Cannot read property 'sections' of undefined

      128 |     }).sections;
      129 | 
    > 130 |     const style = _.find(nestedSections, {
      131 |       reference: 'style'
      132 |     }).sections;
      133 | 
      
      at Object.<anonymous> (tools/gulp/docs/__tests__/nestSections.test.js:130:19)```

Reply:
I'm not getting this error some reason